www/js/app.js

```diff
@@ -1296,6 +1306,23 @@ function handleFileDrop (packet) {
     document.getElementById('archiveFiles').value = null;
 }
 
+document.getElementById('libraryBtn').addEventListener('click', function (e) {
+    e.preventDefault();
+    uiUtil.tabTransitionToSection('library', params.showUIAnimations);
+
+    const libraryContent = document.getElementById('libraryContent');
+    const iframe = libraryContent.contentWindow.document.getElementById('libraryIframe');
+    try {
+        // eslint-disable-next-line no-new-func
+        Function('try{}catch{}')();
+        iframe.setAttribute('src', params.libraryUrl);
+    } catch (error) {
+        const fakeElement = document.createElement('a');
+        fakeElement.setAttribute('href', params.altLibraryUrl);
+        uiUtil.warnAndOpenExternalLinkInNewTab(e, fakeElement);
```

Comment:
Two issues that are minor but could make a big difference to the UX:

1. I don't think it's necessary to ask the user if they wish to open a new tab: just do it. After all, the user has clicked on "Open library", and any user with half a brain will know that the library is an external URL...
2. Do a programmatic click on `btnConfigure` after you launch the new tab, so that the user isn't left staring at a disconcerting blank screen when they close the external tab... (see screenshot below).

![image](https://github.com/kiwix/kiwix-js/assets/4304337/70a6b071-ca0f-4692-ac9d-cd3245dfcb4c)